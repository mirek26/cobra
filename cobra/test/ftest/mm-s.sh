$COBRA_HOME/cobra mm3x3.py > /dev/null;
echo "0 0 3 4 1 1" | $COBRA_HOME/cobra-backend -m s .cobra.in
