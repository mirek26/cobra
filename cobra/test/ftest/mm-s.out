
[1;97m===== SIMULATION =====[0m
Select an experiment: 
0) guess [ A A A ]  - M: 8 0 12 0 0 6 0 0 1 F: 3 18 0 18 18 0 18 18 9 
1) guess [ A A B ]  - M: 1 4 6 3 4 6 0 2 1 F: 9 3 1 5 1 0 18 5 9 
2) guess [ A B C ]  - M: 0 0 3 6 6 6 2 3 1 F: 18 18 0 3 0 0 3 0 9 
> [32mEXPERIMENT: guess A A A [0m
Select an outcome: 
0) 0 + 0 
-) 0 + 1 (unsatisfiable)
2) 1 + 0 
-) 0 + 2 (unsatisfiable)
-) 1 + 1 (unsatisfiable)
5) 2 + 0 
-) 0 + 3 (unsatisfiable)
-) 1 + 2 (unsatisfiable)
8) 3 + 0 
> [32mOUTCOME: 0 + 0
  ->   (Â¬x3A âˆ§ Â¬x3A âˆ§ Â¬x3A âˆ§ Â¬x1A âˆ§ Â¬x2A âˆ§ Â¬x2A âˆ§ Â¬x1A âˆ§ Â¬x1A âˆ§ Â¬x2A)

[0mSelect an experiment: 
1) guess [ A A B ]  - M: 1 3 4 0 0 0 0 0 0 F: 9 5 5 18 18 18 18 18 18 
2) guess [ A B B ]  - M: 1 1 2 0 2 2 0 0 0 F: 9 9 5 18 5 7 18 18 18 
3) guess [ A B C ]  - M: 0 0 2 2 2 2 0 0 0 F: 18 18 3 7 3 7 18 18 18 
4) guess [ B B B ]  - M: 1 0 3 0 0 3 0 0 1 F: 9 18 3 18 18 3 18 18 9 
5) guess [ B B C ]  - M: 0 0 1 1 0 3 0 2 1 F: 18 18 9 9 18 3 18 5 9 
> [32mEXPERIMENT: guess A B C [0m
Select an outcome: 
-) 0 + 0 (unsatisfiable)
-) 0 + 1 (unsatisfiable)
2) 1 + 0 
3) 0 + 2 
4) 1 + 1 
5) 2 + 0 
-) 0 + 3 (unsatisfiable)
-) 1 + 2 (unsatisfiable)
-) 3 + 0 (unsatisfiable)
> [32mOUTCOME: 1 + 1
  ->   ((x1A âˆ§ x3B âˆ§ Â¬x3C âˆ§ Â¬x2B âˆ§ Â¬x2C) âˆ¨ (x1A âˆ§ x2C âˆ§ Â¬x2B âˆ§ Â¬x3C âˆ§ Â¬x3B) âˆ¨ (x3A âˆ§ x2B âˆ§ Â¬x1C âˆ§ Â¬x3C âˆ§ Â¬x1A) âˆ¨ (x1C âˆ§ x2B âˆ§ Â¬x3A âˆ§ Â¬x1A âˆ§ Â¬x3C) âˆ¨ (x2A âˆ...

[0mSelect an experiment: 
1) guess [ A A B ]  - M: 0 1 1 0 0 0 0 0 0 F: 18 9 9 18 18 18 18 18 18 
2) guess [ A A C ]  - M: 0 1 1 0 0 0 0 0 0 F: 18 9 9 18 18 18 18 18 18 
3) guess [ A B B ]  - M: 0 1 0 0 0 1 0 0 0 F: 18 9 18 18 18 9 18 18 18 
6) guess [ B A A ]  - M: 0 1 1 0 0 0 0 0 0 F: 18 9 9 18 18 18 18 18 18 
7) guess [ B A B ]  - M: 0 0 1 0 1 0 0 0 0 F: 18 18 9 18 9 18 18 18 18 
8) guess [ B A C ]  - M: 0 0 0 1 0 1 0 0 0 F: 18 18 18 9 18 9 18 18 18 
9) guess [ B B A ]  - M: 0 0 1 0 1 0 0 0 0 F: 18 18 9 18 9 18 18 18 18 
10) guess [ B B B ]  - M: 0 0 1 0 0 1 0 0 0 F: 18 18 9 18 18 9 18 18 18 
11) guess [ B B C ]  - M: 0 0 0 0 0 1 0 1 0 F: 18 18 18 18 18 9 18 9 18 
12) guess [ B C A ]  - M: 0 0 0 1 0 1 0 0 0 F: 18 18 18 9 18 9 18 18 18 
13) guess [ B C B ]  - M: 0 0 0 0 0 1 0 1 0 F: 18 18 18 18 18 9 18 9 18 
14) guess [ B C C ]  - M: 0 0 0 1 0 0 0 0 1 F: 18 18 18 9 18 18 18 18 9 
> [32mEXPERIMENT: guess A A B [0m
Select an outcome: 
-) 0 + 0 (unsatisfiable)
1) 0 + 1 
2) 1 + 0 
-) 0 + 2 (unsatisfiable)
-) 1 + 1 (unsatisfiable)
-) 2 + 0 (unsatisfiable)
-) 0 + 3 (unsatisfiable)
-) 1 + 2 (unsatisfiable)
-) 3 + 0 (unsatisfiable)
> [32mOUTCOME: 0 + 1
  ->   ((Â¬x3B âˆ§ Â¬x3A âˆ§ Â¬x1B âˆ§ Â¬x1A âˆ§ Â¬x2A âˆ§ Â¬x1A âˆ§ x2A) âˆ¨ (x3A âˆ§ Â¬x3B âˆ§ Â¬x1B âˆ§ Â¬x1A âˆ§ Â¬x2A âˆ§ Â¬x2B âˆ§ Â¬x1A) âˆ¨ (Â¬x3A âˆ§ Â¬x3B âˆ§ Â¬x2A âˆ§ Â¬x1A âˆ§ Â¬x2B âˆ§ x1A âˆ§ ...

[0m[1;97mSOLVED in 3 experiments![0m
Note: there is only one possible code but the last outcome is not marked as final.
TRUE: x1B x2C x3C 
FALSE: x1A x1C x2A x2B x3A x3B 

